src/Core/SettingsChangesHistory.h

```diff
@@ -93,6 +93,8 @@ static std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges> sett
               {"input_format_hive_text_allow_variable_number_of_columns", false, true, "Ignore extra columns in Hive Text input (if file has more columns than expected) and treat missing fields in Hive Text input as default values."},
               {"query_plan_optimize_lazy_materialization", true, true, "Add new setting to use query plan for lazy materialization optimisation"},
               {"query_plan_max_limit_for_lazy_materialization", 10, 10, "Add new setting to control maximum limit value that allows to use query plan for lazy materialization optimisation. If zero, there is no limit"},
+              {"first_day_of_week", "Monday", "Monday", "Added a setting for the first day of the week for date/time functions"},
+              {"temporary_data_in_cache_reserve_space_wait_lock_timeout_milliseconds", (10 * 60 * 1000), (10 * 60 * 1000), "Wait time to lock cache for sapce reservation in temporary data in filesystem cache"},
     }},
```

Comment:
>sapce

typo

I would use `Cache lock timeout to reserve temporary data space in the file system cache`